Exercism/Gigasecond/Gigasecond.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercism.Gigasecond
+{
+    public static class Gigasecond
+    {
+        public static DateTime Date(DateTime birthDate)
```

Comment:
I agree. Again that all came premade so I'm just using what they gave me to solve the tests. Though since they don't check anything I'll go ahead and rename it.